lib/chef/telemetry/decision/argument.rb

```diff
@@ -0,0 +1,37 @@
+class Chef
+  class Telemetry
+    class Decision
```

Comment:
I like the `Decision` class name, I think it is a little more clear than my usage of `Strategy`. `Strategy` is an overloaded, pattern-ey thing in my head.